### PR TITLE
Ops 883 create project front end

### PR DESCRIPTION
**Pull request by xlorepdarkhelm**
## What changed

Added "Create Project" page.

## Issue

[#883 ](https://github.com/HHS/OPRE-OPS/issues/883)

## How to test

1. Open site & log in.
2. Go to Create Project.
3. Do stuff.

## Screenshots

image

### CODE REVIEW

File: frontend/src/api/postProjects.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import ApplicationContext from "../applicationContext/ApplicationContext";
+
+export const postProject = async (item) => {
+    const api_version = ApplicationContext.get().helpers().backEndConfig.apiVersion;
+
+    const newProject = { ...item };
+
+    // TODO: Something something project type.
```
Comment (johndeange) [inline frontend/src/api/postProjects.js:8]: Is there something that needs to be done here still?

File: frontend/src/pages/projects/CreateProject.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import App from "../../App";
+import { useSelector, useDispatch } from "react-redux";
+import ProjectTypeSelect from "./ProjectTypeSelect";
+import { setProjectId, setProjectShortTitle, setProjectTitle, setProjectDescription } from "./createProjectSlice";
+import { postProject } from "../../api/postProjects";
+
+export const CreateProject = () => {
+    const [formData, setFormData] = React.useState({});
```
Comment (johndeange) [inline frontend/src/pages/projects/CreateProject.jsx:9]: see linting error

**Review by johndeange — APPROVED** [review]

File: frontend/src/api/postProjects.js (hunk 1 above)
Comment (xlorepdarkhelm) [reply frontend/src/api/postProjects.js:8]: Yes, but after our demo. Basically we're not sending/storing the project type or using it at all on the backend. We only know of Research projects. So... Tim and I went with just skipping actually building that for now.

File: frontend/src/pages/projects/CreateProject.jsx (hunk 2 above)
Comment (xlorepdarkhelm) [reply frontend/src/pages/projects/CreateProject.jsx]: Should be resolved now.

**Pull request merged**